src/cli/undeploy/bundle/UndeployBundle.definition.ts

```diff
@@ -0,0 +1,50 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright IBM Corp, 2019
+*
+*/
+
+import { ICommandDefinition } from "@brightside/imperative";
+import { NameOption } from "../../shared/Name.option";
+import { CicsDeployProfileOption } from "../../shared/CicsDeployProfile.option";
+import { CicsplexOption } from "../../shared/Cicsplex.option";
+import { ScopeOption } from "../../shared/Scope.option";
+import { CsdgroupOption } from "../../shared/Csdgroup.option";
+import { ResgroupOption } from "../../shared/Resgroup.option";
+import { TimeoutOption } from "../../shared/Timeout.option";
+
+/**
+ * Imperative command for the Bundle sub-option of Deploy.
+ *
+ */
+export const UndeployBundleDefinition: ICommandDefinition = {
+    name: "bundle",
+    aliases: ["b", "bun", "bund"],
+    summary: "Deploy a CICS bundle",
+    description: "Undeploy a CICS bundle from a CPSM managed group of CICS regions. " +
+                 "The DFHDPLOY utility is used to undeploy and remove a Bundle resource " +
```

Comment:
```suggestion
                 "The DFHDPLOY utility is used to undeploy and remove a BUNDLE resource " +
```

Reply:
I took that text from the existing KC documentation for DFHDPLOY... but I do agree with you, I hate how the KC tends to use 'Bundle' or 'bundle' where 'BUNDLE' would be clearer.

Reply:
I'm wrong... just checked the doc... it's all lower case there.